Makefile

```diff
@@ -184,16 +187,14 @@ clone::
 ###############################################################################
 #                                DEPLOYMENT                                   #
 ###############################################################################
-deploy-develop::
-	@bin/dep deploy develop -vv --revision="develop"
+deploy-development::
```

Comment:
Convention across all projects has been to call this target `develop` as derived from the naming convention of git flow (https://nvie.com/posts/a-successful-git-branching-model/). I'd much appreciate to keep it that way.

Reply:
I don't see deployment targets related to git flow branch names. If this was about deploying the develop branch $somewhere I'd agree, but I see this as deploying $something to the development server

Reply:
Let's discuss this in Codeworks and have a vote on how to proceed with this convention.

Reply:
The decision was to use the terms "develop", "staging" and "master" for branches, targets clone presets etc.